### PR TITLE
making changes

### PR DESCRIPTION
**Pull request by njkmr0027**

### CODE REVIEW

File: src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown [hunk 1]
```diff
@@ -666,3 +666,41 @@ Credit Card Vendor|-|USD 284
 [**Standard Accounting Extract**](./v4.financial-integration-use-cases-extract-information.html#eccccepno)
 
 [**JSON Example**](json/FI-API-ReportID-7F00502973D84DEFA49E-Example-6.2.4.json)
+
+### The use case examples below illustrate Funds and Grants Integration (FGI) scenarios and the related records of Funds Reservation and Funds Check. The use case documentation is not meant to be a comprehensive list of all use cases and FGI scenarios. However, the JSON from the Financial Integration Service will provide all the required data to support all potential financial use cases.
+
```
Comment (RoAnnC1) [inline src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown:671]: @njkmr0027 You have this styled as a heading, but it's really just a paragraph. Does it need to be bold? Could we style it as a note? Let me know and I can fix the formatting.

File: src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown [hunk 2]
```diff
@@ -666,3 +666,41 @@ Credit Card Vendor|-|USD 284
...
+#### Simple Funds Reservation 
+#### Funds Reservation on Request 
+
+
+##### Description
+   When the Request is created and FGI is enabled for a Group that an employee belongs to, during the workflow, when the Funds Reservation step is reached, the workflow is paused, an event ___ is published and a Request Obligation document is made available to financial system.
```
Comment (RoAnnC1) [inline src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown:677]: What is the long ___ for here?

File: src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown [hunk 3]
```diff
@@ -666,3 +666,41 @@ Credit Card Vendor|-|USD 284
...
+
+The Financial system attempts to create a commitment against the details in the Request Obligation.
+If Successful/unsuccessful,  the financial system responds with a message.
+
+[**Sample Confirmation Payload**](json/FI-FundsReservation-Cnf-57e4d6c30f2c4d5bbd4ac7adc8bc8233-Example-1.0.json)
+
+#### Release Request Commitment:
+##### When a Request is closed,  FGI evaluates the status of all linked Reports.
+•	When all the linked Reports are paid, release of Request commitment is initiated.
+•	A Request Release Payload is made available to the financial system and the event ___ is published
```
Comment (RoAnnC1) [inline src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown:703]: Same comment as for line 677.

**Review by RoAnnC1 — CHANGES_REQUESTED** [review]
@njkmr0027 Have a couple of questions. Once you answer or resolve I'll merge this and do a little editing.

File: src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown (hunk 3 above)
Comment (njkmr0027) [reply src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown]: updated the document

**Review by RoAnnC1 — APPROVED** [review]

**Pull request merged**